### PR TITLE
denies gzipped response from auth provider (vitae)

### PR DESCRIPTION
**Pull request by dfinninger**

### CODE REVIEW

**Comment by brasic** [comment]
Hi!  Not sure if you guys care about this being an open source package anymore, but hardcoding the Accept-Encoding header to a specific value might not be friendly for people using this outside PeopleAdmin, if there is anyone.  I would recommend instead yielding the headers to a caller-controlled configurator in the same way `modify_connection` does.

**Comment by dfinninger** [comment]
@brasic Whoops! Forgot this was open. We wound up not trying to push this through.
Thanks for the heads up.

**Comment by rpond-pa** [comment]
Get out of here you non-PA Chicagoian.  :P  !!!

**Comment by mmorast** [comment]
What up Carl! How is Chi-town?

**Comment by rpond-pa** [comment]
OMG this PR is a chat channel for old PA alumni.  rolles :eyes:

**Comment by mmorast** [comment]
Devon!! How are you doing?

**Comment by dfinninger** [comment]
MMonty! Doing alright! Just rocking it over here. How're you?

**Comment by mmorast** [comment]
Livin' the homeaway dream! I am doing well...

**Comment by brasic** [comment]
Hey @mmorast, loving it here, hope your new gig is going well!

**Comment by rpond-pa** [comment]
Hey @brasic, let's get Taco Deli some time!  Oh wait.  :cry:

File: lib/omniauth-vitae/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Omniauth
   module Vitae
-    VERSION = '0.0.2'
+    VERSION = '0.1.0'
```
Comment (lwoodson) [inline lib/omniauth-vitae/version.rb:3]: Should be double quotes :bike: :house:

**Comment by lwoodson** [comment]
Hey PA peeps!  Hope all is well.

**Comment by dfinninger** [comment]
It's like the Git Logs are coming to life in here...

**Comment by mdespuits** [comment]
Well I think I need to join the party too! Hey all! :smile:

**Comment by rpond-pa** [comment]
Who we forgetting?

**Comment by jjasghar** [comment]
To quote a PR that just got merged in Chef land: 
:+1: :cake: :+1: :rocket: :rocket: :rocket: :rocket: :rocket: :rocket:

**Comment by rpond-pa** [comment]
:rocket: :hankey:

**Comment by jjasghar** [comment]
:docker: :docker: :docker:

**Comment by dfinninger** [comment]
^^^ @rpond-pa 's favorite emoji combo these days...

File: lib/omniauth-vitae/version.rb (hunk 1 above)
Comment (billeisenhauer) [inline lib/omniauth-vitae/version.rb:3]: I like the bike-shed symbology!

File: lib/omniauth/strategies/vitae.rb [hunk 2]
```diff
@@ -29,9 +29,10 @@ class Vitae < OmniAuth::Strategies::OAuth2
       def raw_info
         @raw_info ||= begin
           modify_connection
-          access_token.get('/api/users/self',
-                           :headers => { 'Accept' => 'application/json' }).
-            parsed.fetch('results').first
+          access_token.get('/api/users/self', :headers => {
```
Comment (jjasghar) [inline lib/omniauth/strategies/vitae.rb:32]: I think `:headers` should be on it's own line, then the following lines 33-34 spaced in.

**Comment by mmorast** [comment]
I am sure we can fit a couple more in these comments

File: lib/omniauth-vitae/version.rb (hunk 1 above)
Comment (jjasghar) [inline lib/omniauth-vitae/version.rb:3]: @lwoodson you know rubocop'll yell at you for that. RUBOCOP ALL THE THINGS

**Comment by billeisenhauer** [comment]
Worked from home last 8 out of 9 days.  Presently at a coffee shop working (how is this even possible?!).  Pushed 3 PRs today(!) to production simply by pushing to master branch.

And...I'm not wearing pants.

How's it going PA-ers?

File: lib/omniauth-vitae/version.rb (hunk 1 above)
Comment (mmorast) [inline lib/omniauth-vitae/version.rb:3]: Someone ran rubocop?!?!?!?

**Comment by ChloeCoon** [comment]
Hello everyone, @billeisenhauer I appreciate that the first thing I read was about you being pantsless.

**Comment by lwoodson** [comment]
Presently at a coffee shop not wearing pants @billeisenhauer?  That's BOLD. :D

**Comment by dfinninger** [comment]
@billeisenhauer Well, I was working with syed and randy until 10ish last night... but I was at home, so does that count?

**Comment by mmorast** [comment]
This just got weird! I like it!

**Comment by ChloeCoon** [comment]
Hi @mmorast ! Long time no chat!

**Comment by mmorast** [comment]
Chloe! I know. I kept missing any outings that you would been at.

**Comment by ChloeCoon** [comment]
I probably wasn't there so you wouldn't have missed anything

File: lib/omniauth-vitae/version.rb (hunk 1 above)
Comment (dfinninger) [inline lib/omniauth-vitae/version.rb:3]: only on our Chef code, lol

**Comment by cbradbury** [comment]
so I click merge since things reached 'weird', right?

**Comment by rpond-pa** [comment]
Poor Carl.  He was trying to be helpful and then got like 18 thousand replies.

**Comment by rpond-pa** [comment]
Hey @cbradbury !  Welcome to Slack 3.0

File: lib/omniauth-vitae/version.rb (hunk 1 above)
Comment (jjasghar) [inline lib/omniauth-vitae/version.rb:3]: Nice! Now you need chefspec :+1:

Comment (dfinninger) [inline lib/omniauth-vitae/version.rb:3]: Already done :+1:

**Comment by joshuaflanagan** [comment]
The build is failing!

(hi all)

File: lib/omniauth-vitae/version.rb (hunk 1 above)
Comment (jjasghar) [inline lib/omniauth-vitae/version.rb:3]: Or go deeper: https://github.com/openstack/cookbook-openstack-common/blob/master/TESTING.md

**Comment by mmorast** [comment]
JOSH!!

**Comment by jjasghar** [comment]
@joshuaflanagan I'll fix the dashboard....gotta reset the rasberry pi....

**Comment by billeisenhauer** [comment]
@lwoodson Its a special kind of coffee shop.
@joshuaflanagan may not be wearing pants either, just sayin'.

Hey @mmorast!
Hey @ChloeCoon!  Keeping an eye on @mattdbridges?

**Comment by ChloeCoon** [comment]
@billeisenhauer, hey there! Pretty sure he's keeping an eye on me.

**Comment by mmorast** [comment]
@billeisenhauer Sounds like you are being productive and writing code!

@ChloeCoon keeps everyone in line...

**Comment by ChloeCoon** [comment]
@mmorast Not anymore, I no longer have to yell at people about workflows and versioning.

**Comment by ChloeCoon** [comment]
Or whatever things I used to yell about.

**Comment by rpond-pa** [comment]
[ starts to list things @ChloeCoon yelled about ] 
Hold on, gonna need some pen and paper for this one.

**Comment by ChloeCoon** [comment]
Let's meet in the conference room and whiteboard it, should only take 8 hours or so.

**Comment by mmorast** [comment]
The list is long and distinguished... @ChloeCoon is in a better place.

**Comment by brasic** [comment]
@ChloeCoon In honor of you I'm yelling at my dogs about workflows and versioning.

**Comment by ChloeCoon** [comment]
aww Henry and Lucy!!! I miss those guys @brasic

**Comment by rpond-pa** [comment]
Sweet!  A few more comments and we will have surpassed EEO overhaul comments.  

pull_requests_ _peopleadmin_portals

**Comment by jjasghar** [comment]
image

**Comment by lwoodson** [comment]
I'll see you and raise you a wholly inappropriate Cosby jpeg.

cosby

**Comment by dfinninger** [comment]
@jjasghar is that where you got inspiration for your profile pic?

**Comment by jjasghar** [comment]
@dfinninger actually no, i did it randomly, but i've been finding random pics like that around the internet.

**Comment by joshuaflanagan** [comment]
Oh my, he went to EEO Overhaul. Trying to drag @ntl into this?

**Comment by mmorast** [comment]
Thanks Josh! Now I know who I was missing in this conversation.

**Comment by rpond-pa** [comment]
That was the plan, @joshuaflanagan.  @ntl and @gaijin1973 too.

**Comment by lwoodson** [comment]
But that's just one generation of PA devs.  What about @rubyminer?

**Comment by cbradbury** [comment]
I hope I will see some of you lovely people at LoneStarRuby.

**Comment by jjasghar** [comment]
@cbradbury i'm really hoping i get picked to talk :D

**Comment by ChloeCoon** [comment]
@cbradbury That will be across the street from my office, we should meet for lunch!

**Comment by barrettb-zz** [comment]
What is all this racket?!

**Comment by rpond-pa** [comment]
Where's @manewitz?!  Sup @barrettb!

**Comment by manewitz** [comment]
slj

**Comment by ntl** [comment]
Please don't disturb me with mentions, I'm tying off on a PR for the final stage of the EEO refactor. I should have it submitted soon.

**Comment by barrettb-zz** [comment]
_QA rejects EEO refactor_

**Comment by manewitz** [comment]
`> Nathan.computed_ethnicity`
:trollface:

**Comment by jjasghar** [comment]
The irony of this was today was my first call with rackspace as a Chef employee today. I'm having flashbacks reading through this thread.

**Comment by ntl** [comment]
_Sneaks in a change to Object Momma that causes all acceptance tests to randomly start failing. QA too distracted to notice EEO refactor is forced pushed to production._

(Actually is just upgrade to capybara/phantomjs)

**Comment by lwoodson** [comment]
Don't feel bad, @ntl.  I'll have reporting wrapped up in about 2 weeks.

**Comment by rpond-pa** [comment]
And I think @mattdbridges is waiting on code reviews for Asset Pipeline .

**Comment by rpond-pa** [comment]
Closing this PR.  Not enough comments.

**Comment by mdespuits** [comment]
I'm hearing docker docker docker but the asset pipeline is still not a thing? @rpond-pa I know you can make that happen.

**Comment by billeisenhauer** [comment]
There is no asset pipeline in a Spring Java app.

**Comment by brasic** [comment]
Bwahahahah

**Comment by gaijin1973** [comment]
I don't think I'm witty enough to contribute to this comment thread. However, if anyone wants to reopen all the going-away-present PR's I opened right before I left and give me repo access, I'm happy to riff away there. ;-)

**Comment by lwoodson** [comment]
lol @billeisenhauer

**Comment by praveenpinto** [comment]
@jjasghar We've since moved from raspberry pi to mac mini with 60 inch tv screens!!
freshprincedance

**Comment by mmorast** [comment]
I still could not log into it half the time, but the charts load faster! Yep confirmed, I can not log in.

**Pull request closed without merging**